### ISSUE TITLE
Add option to hide "Single user page" in JetEngine profile builder

### ISSUE THREAD

**Issue by LevelUpGit** [CrocoBlock/suggestions#6603]
Add option to hide "Single user page" in JetEngine profile builder per roles just like other subpages.

**Comment by morosmo** [comment]
+1

**Comment by Lonsdale201** [comment]
+1

**Comment by LevelUpGit** [comment]
To expand on this:

We have a project where we have Editors on frontend that need to see user profiles and edit them if needed. Only way we found we can do this is via "Single user page" in profile builder, but there is an a issue, it is publicly available automatically, and there is no option to make it private. We found a workaround by setting Single User Page as private, but we don't like this solution.

Basically for User Page there should be a filed to show it to only specific roles or to be a public profile.

**Comment by morosmo** [comment]
+1
Crocoblock need to focus on profile builder as this is the key to create multivendor sites using jetengine

**Comment by aalexandr21** [comment]
was added and released in JetEngine 3.8.7

<img width="1729" height="855" alt="Image" src="https://github.com/user-attachments/assets/ffbdc889-fd13-44e8-8f51-e19bfbf19b77" />
